sites/platform/src/create-apps/app-reference.md

```diff
@@ -63,8 +139,19 @@ Available languages and their supported versions:
 
 These are used in the format `runtime:version`:
 
-{{< readFile file="registry/images/examples/full/php.app.yaml" highlight="yaml" configFile="app" >}}
+{{% version/specific %}}
+```yaml {configFile="app"}
+type: 'php:{{% latest "php" %}}'
+```
+<--->
+```yaml {configFile="app"}
+applications:
+    myapp:
+        type: 'php:{{% latest "php" %}}'
+```
+{{% /version/specific %}}
 
+{{< version/specific >}}
 ## Sizes
```

Comment:
I believe the Sizes section should be removed since we don't have plans anymore and project resources are allocated through container profiles and `resources:set`?

Reply:
Should already be in a vendor/specific catch starting line 185 to use resources

https://docs.upsun.com.apps-context-6oqlwzy-652soceglkw4u.eu-3.platformsh.site/create-apps/app-reference.html#resources